config/locales/en.yml

```diff
@@ -560,14 +559,26 @@ en:
-      immigration_header: Immigration (Upper Tribunal)
+      immigration_header_certificated: Immigration (Upper Tribunal)
       immigration_certificated_paragraph_one: This must be for legal representation for an immigration matter in the Immigration and Asylum Chamber of the Upper Tribunal including in relation to an appeal or review from the Immigration and Asylum Chamber of the First-tier Tribunal.
       immigration_certificated_paragraph_two_html: Guidance for this is in paragraphs 25 to 29, 32(1), 32A(1) of Part 1 of Schedule 1 to the <a href='https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1045682/Lord_Chancellor_s_guidance_on.pdf' target='_blank' rel='noreferrer noopener'>Legal Aid, Sentencing and Punishment of Offenders Act 2012</a> (LASPO).
-      asylum_header: Asylum (Upper Tribunal)
+      asylum_header_certificated: Asylum (Upper Tribunal)
       asylum_certificated_paragraph_one: This must be for legal representation for an asylum matter in the Immigration and Asylum Chamber of the Upper Tribunal including in relation to an appeal or review from the Immigration and Asylum Chamber of the First-tier Tribunal
       asylum_certificated_paragraph_two_html: Guidance for this is in paragraphs 24 to 30, 32(1) and 32A(1) of Part 1 of Schedule 1 to the <a href='https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1045682/Lord_Chancellor_s_guidance_on.pdf' target='_blank' rel='noreferrer noopener'>Legal Aid, Sentencing and Punishment of Offenders Act 2012</a> (LASPO).
       domestic_abuse_header: Domestic abuse
       domestic_abuse_paragraph: This must be for legal representation for a domestic abuse matter (injunction or other order for protection from harm, or committal for breach), FGM protection order, or forced marriage protection order.
+      immigration_and_asylum_controlled_paragraph: These must be for matters in the Immigration and Asylum Chamber of the First-tier Tribunal.
+      immigration_header_controlled: Legal representation for an immigration matter
+      immigration_controlled_paragraph_one: This must be for legal representation for an immigration matter in the Immigration and Asylum Chamber of the First-tier Tribunal.
+      immigration_controlled_paragraph_two_html: Guidance for this is in paragraphs 25 to 29, 32(1), 32A(1) of Part 1 of Schedule 1 to the <a href='https://www.legislation.gov.uk/ukpga/2012/10/schedule/1' target='_blank' rel='noreferrer noopener'>Legal Aid, Sentencing and Punishment of Offenders Act 2012</a> (LASPO).
+      asylum_header_controlled: Legal help for an immigration or asylum matter, or help at court or legal representation for an asylum matter
+      asylum_controlled_paragraph_one: "This must be for a matter in the Immigration and Asylum Chamber of the First-tier Tribunal, covering:"
```

Comment:
should these quotation marks be here?

Reply:
Very good question, and the answer is yes, and the reason is because YAML isn't as smart as we'd like. YAML is generally smart enough to know that `foo: bar` means `bar` is a string, i.e. `"bar"`. Because of that we generally write `foo: bar` instead of the more explicit `foo: "bar"` because it's less punctuation to type.

BUT where strings contain special characters like `:`, YAML isn't sure if the strings are strings or if they're actually something else, so it freaks out.

So that's a long way of saying, because of the `:` at the end of the line, we have to wrap the text in quotation marks!